src/stores/Comment/Comment.js

```diff
@@ -39,7 +42,39 @@ export const Comment = types.model("Comment", {
       }
     }
 
+    function setEditMode( newMode ) {
+      self.isEditMode = newMode;
+    }
+
+    function setDeleted( newMode ) {
+      self.isDeleted = newMode;
+    }
+
+    function setConfirmMode( newMode ) {
+      self.isConfirmDelete = newMode;
+    }
+
+    async function updateComment( comment ) {
+      await self.sdk.invoke("comments:update", {
+        id: self.id,
+        text: comment,
+      });
+      self.setEditMode(false);
+    }
+
+    async function deleteComment() {
+      await self.sdk.invoke("comments:delete", {
+        id: self.id,
+      });
+      self.setDeleted(true);
```

Comment:
Probably should reset the confirmation as well here.

```suggestion
      self.setDeleted(true);
      self.setConfirmMode(false);
```